plexpy/api.py

```diff
@@ -13,86 +16,143 @@
 #  You should have received a copy of the GNU General Public License
 #  along with PlexPy.  If not, see <http://www.gnu.org/licenses/>.
 
-from plexpy import db, cache, versioncheck, logger, helpers
-
+from plexpy import versioncheck, logger, plextv, pmsconnect, datafactory, graphs, users
+import os
 import plexpy
 import json
-from xml.dom import minidom
+import traceback
+import cherrypy
+import re
+import hashlib
+import random
+import xmltodict
 
-cmd_list = ['getHistory', 'getLogs', 'getVersion', 'checkGithub', 'shutdown', 'restart', 'update']
+cmd_list = ['getLogs', 'getVersion', 'checkGithub', 'shutdown',
+            'getSettings', 'restart', 'update', 'getApikey', 'getHistory',
+            'getMetadata', 'getUserips', 'getPlayby, getSync']
```

Comment:
Shouldn't this be: 'getPlayby', 'getSync'? It looks like you are missing a comma.